lib/legacy/class.html

```diff
@@ -373,6 +350,11 @@
         Polymer.LegacyElementMixin(HTMLElement));
       // decorate klass with registration info
       klass.is = info.is;
+      // if user provided template on info, make sure the static _template
+      // is set so the static template getter uses it
+      if (info._template !== undefined) {
```

Comment:
Add a test that sets a template on a behavior.

Reply:
Done, in test/unit/behaviors.html.